includes/amp-helper-functions.php

```diff
@@ -579,23 +579,56 @@ function amp_get_schemaorg_metadata() {
 		),
 	);
 
-	/**
```

Comment:
This section is only moved to [line 606](https://github.com/Automattic/amp-wp/pull/1144/files#diff-5e173f2331b78ccd3da05258383dcc55R606), mainly unchanged.